cmd/notary/integration_test.go

```diff
@@ -245,6 +247,153 @@ func TestClientTUFInteraction(t *testing.T) {
 	require.False(t, strings.Contains(string(output), target))
 }
 
+func TestClientDeleteTUFInteraction(t *testing.T) {
+	// -- setup --
+	setUp(t)
+
+	tempDir := tempDirWithConfig(t, "{}")
+	defer os.RemoveAll(tempDir)
+
+	server := setupServer()
+	defer server.Close()
+
+	tempFile, err := ioutil.TempFile("", "targetfile")
+	require.NoError(t, err)
+	tempFile.Close()
+	defer os.Remove(tempFile.Name())
+
+	// Setup certificate
```

Comment:
Non-blocking nitpick - setting up the cert and adding a delegation role + target to the delegation seems to also be done by `TestPurge` and `TestClientDelegationsInteraction` and `TestClientDelegationsPublishing`, although this one doesn't need the key ID of the cert.  I was wondering if we could factor this logic out to a helper function?